### PR TITLE
[13.0][MIG] portal_partner_select_all: Migration to 13.0

### PR DESCRIPTION
**Pull request by skeller1**
**Standard migration to V13:**

portal_partner_select_all_v13

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

File: portal_partner_select_all/i18n/portal_partner_select_all.pot [hunk 1]
```diff
@@ -14,17 +14,18 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: portal_partner_select_all
-#: model:ir.ui.view,arch_db:portal_partner_select_all.portal_wizard
+#: model_terms:ir.ui.view,arch_db:portal_partner_select_all.portal_wizard
 msgid "<i class=\"fa fa-warning\"/> You're about to invite to portal all the contacts list below."
```
Comment (mimusica) [inline portal_partner_select_all/i18n/portal_partner_select_all.pot]: "You're about to invite all contacts listed below to the portal."

**Review by mimusica — CHANGES_REQUESTED** [review]

File: portal_partner_select_all/i18n/portal_partner_select_all.pot (hunk 1 above)
Comment (skeller1) [reply portal_partner_select_all/i18n/portal_partner_select_all.pot]: Changed it, thank you

**Comment by mimusica** [comment]
YW :-)

Met vriendelijke groeten, With kind regards, Cordialement, Atentamente,

Christophe Langenberg

Op wo 29 jul. 2020 om 12:13 schreef Stephan Keller <notifications@github.com

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by HaraldPanten** [comment]
patch here for merging, @rousseldenis ?

**Comment by skeller1** [comment]
Ready for Merge

Thank you

**Pull request merged**